### DISCUSSION TITLE
how can I require interactive confirm to wait to <enter> key ?

### DISCUSSION THREAD

**Discussion by elfrucool** [pterm/pterm#643] [Q&A] [answered]
for some people is confusing that confirm dialog moves next by a single keystroke without typing `<enter>` it would be nice to have that option, something like `pterm.DefaultInteractiveConfirm.WithWaitReturn(true).Show()`

**Comment by MarvinJWendt** [comment, marked as answer]
Hi @elfrucool, you can make a feature request for this. I agree that the behavior sometimes feels weird.

https://github.com/pterm/pterm/issues/new?assignees=&labels=proposal&projects=&template=feature_request.md&title=
